tests/python/unittest/test_loss.py

```diff
@@ -64,7 +64,9 @@ def get_net(num_hidden, flatten=True):
     fc3 = mx.symbol.FullyConnected(act2, name='fc3', num_hidden=num_hidden, flatten=flatten)
     return fc3
 
-@with_seed(1234)
+# @lanking520: adding Xavier initializer and problem solved
```

Comment:
I think you can remove this comment as people in future won't understand that it was related to flaky tests

Reply:
@haojin2 shall we do that? Or I can change to something clearer: `because of missing Xavier initializer, this test used to be flaky`

Reply:
Maybe we can move this to the description section of your PR, and eliminate from the code.